firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionGenerator.kt

```diff
@@ -20,23 +20,21 @@ import java.util.UUID
 
 /**
  * [SessionDetails] is a data class responsible for storing information about the current Session.
- *
- * @hide
  */
 internal data class SessionDetails(
   val sessionId: String,
   val firstSessionId: String,
   val sessionIndex: Int,
+  val sessionTimestampUs: Long,
```

Comment:
Let's call it `sessionStartTimestampUs` for clarity sake.

Reply:
Done.